src/main/java/net/neoforged/neoforge/network/PacketDistributor.java

```diff
@@ -5,85 +5,84 @@
 
 package net.neoforged.neoforge.network;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 import java.util.function.BiFunction;
 import java.util.function.Consumer;
+import java.util.function.Function;
 import java.util.function.Supplier;
+
+import com.google.common.collect.Lists;
 import net.minecraft.client.Minecraft;
-import net.minecraft.network.Connection;
 import net.minecraft.network.protocol.Packet;
+import net.minecraft.network.protocol.PacketFlow;
+import net.minecraft.network.protocol.common.ClientboundCustomPayloadPacket;
+import net.minecraft.network.protocol.common.ServerboundCustomPayloadPacket;
+import net.minecraft.network.protocol.common.custom.CustomPacketPayload;
+import net.minecraft.network.protocol.game.ClientGamePacketListener;
+import net.minecraft.network.protocol.game.ClientboundBundlePacket;
 import net.minecraft.resources.ResourceKey;
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.server.level.ServerChunkCache;
 import net.minecraft.server.level.ServerPlayer;
 import net.minecraft.world.entity.Entity;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.level.chunk.LevelChunk;
-import net.neoforged.neoforge.network.simple.SimpleChannel;
 import net.neoforged.neoforge.server.ServerLifecycleHooks;
 
 /**
  * Means to distribute packets in various ways
- *
- * @see SimpleChannel#send(PacketTarget, Object)
- *
- * @param <T>
  */
 public class PacketDistributor<T> {
     /**
      * Send to the player specified in the Supplier
```

Comment:
This javadoc needs to be updated as it no longer is in a supplier (as well as the javadocs for the other distributors)